src/ui/styles/apps/file-preview.css

```
@@ -428,6 +1029,17 @@
   border-radius: 10px;
 }
 
+.markdown-doc a {
+  color: inherit;
+  text-decoration: underline;
+  text-decoration-color: color-mix(in srgb, var(--text-secondary) 45%, transparent);
+  text-underline-offset: 0.18em;
+}
+
+.markdown-doc a:hover {
+  text-decoration-color: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)